docs/GLOSSARY.md

```diff
@@ -0,0 +1,19 @@
+# Glossary
+
+<dl>
+  <dt>
+    portlet
+  </dt>
+  <dd>
+    Portlets are pluggable user interface software components that are managed and displayed in a web portal.
+    uPortal uses the [JSR 168 and JSR 286 portlet specifications](https://en.wikipedia.org/wiki/Java_Portlet_Specification).
+  </dd>
+
+  <dt>
+    soffit
+  </dt>
+  <dd>
+    Soffit is a technology for creating content that runs in Apereo uPortal. It is intended as an alternative to JSR-286 portlet development.
```

Comment:
Which is it, is `Soffit` a technology or a strategy or a content object?

Reply:
Soffit is both a protocol for services interacting with the portal and a Java implementation of that protocol.
@drewwills may be able to add more insights on the topic, as the author of soffits.